SETUP.md

```diff
@@ -62,53 +62,41 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
   1. (You can reinstall with your updated version after you clone the repository if necessary) Reinstall node_modules `cd apps; yarn; cd ..`
 1. (El Capitan) Ensure that openssl is linked: `brew link --force openssl`
 
-
-### Ubuntu 14.04
+### Ubuntu 17.04 ([Ubuntu 17.04 iso image](http://releases.ubuntu.com/17.04/ubuntu-17.04-desktop-amd64.iso?_ga=2.206594387.1901865802.1501006113-173469841.1498786046))
```

Comment:
Suggestion: Deduplicate this URL [using a reference link][about-reference-links].

[about-reference-links]: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

```
Suggestion: Deduplicate this URL [using a reference link][about-reference-links].

[about-reference-links]: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links
```

Probably also could shorten `Ubuntu 17.04 (Ubuntu 17.04 iso image)` to something like `Ubuntu 17.04 (iso)`.